app/src/main/AndroidManifest.xml

```diff
@@ -2,78 +2,73 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="org.kinecosystem.kinit">
 
-    <uses-permission android:name="android.permission.INTERNET"/>
-    <uses-permission android:name="android.permission.READ_PHONE_STATE"/>
-    <uses-permission android:name="android.permission.VIBRATE"/>
+    <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.READ_PHONE_STATE" />
+    <uses-permission android:name="android.permission.VIBRATE" />
 
     <application
+        android:name=".KinitApplication"
         android:allowBackup="true"
         android:icon="@mipmap/ic_launcher_app"
         android:label="@string/app_name"
-        android:name=".KinitApplication"
         android:roundIcon="@mipmap/ic_launcher_app"
         android:supportsRtl="false"
         android:theme="@style/AppTheme">
         <activity
-            android:launchMode="singleInstance"
             android:name=".view.MainActivity"
-            android:screenOrientation="portrait">
-        </activity>
+            android:launchMode="singleInstance"
+            android:screenOrientation="portrait"></activity>
```

Comment:
all other activities ended as `/>` and not with that tag name close like `</activity>`, change it to be the same.